news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: We are working on a new front office theme
```

Comment:
In 2028, this title will be outdated. Maybe we can improve it so that if people find it through Google they understand the context?

Suggestions
- A new front office theme is being built as a successor for Classic Theme
- What comes after Classic Theme
- It's 2022 we need to build a new front office theme

Reply:
Probably the last one fits best, the first will also be outdated

Reply:
But I'm not that happy with both of them :/

Reply:
I'm welcoming suggestions

Wdyt about the mix: "It's 2022 we are working on a new front office theme"

Reply:
What about `We began to work on a new front-office theme`? 

Or your mix is fine to me